packages/forma-36-react-components/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -188,43 +188,68 @@ export const Overview: Story = () => (
 
     <Default />
 
-    <Flex marginBottom="spacingS" marginTop="spacingM">
-      <SectionHeading element="h3">Dropdown default open</SectionHeading>
+    <Flex flexDirection="column" style={{ marginBottom: '200px' }}>
+      <Flex marginBottom="spacingS" marginTop="spacingM">
+        <SectionHeading element="h3">Dropdown default open</SectionHeading>
+      </Flex>
+
+      <Dropdown
+        isOpen
+        isAutoalignmentEnabled
+        usePortal
+        position="bottom-left"
+        toggleElement={
+          <Button size="small" buttonType="muted" indicateDropdown>
+            Choose more options and settings
+          </Button>
+        }
+      >
+        <DropdownList>
+          <DropdownListItem isTitle>Entry Title</DropdownListItem>
+          <DropdownListItem onClick={action('onClick Element')}>
+            Embed existing entry
+          </DropdownListItem>
+          <Dropdown
+            position="right"
+            submenuToggleLabel="Create and embed existing entry"
+          >
+            <DropdownList>
+              <DropdownListItem onClick={action('submenu click')}>
+                Embed as inline element
+              </DropdownListItem>
+              <DropdownListItem isDisabled>
+                Embed as block element
+              </DropdownListItem>
+            </DropdownList>
+          </Dropdown>
+        </DropdownList>
+        <DropdownList border="top">
+          <DropdownListItem>
+            <TextLink href="http://google.com">This is a Link</TextLink>
+          </DropdownListItem>
+        </DropdownList>
+      </Dropdown>
+    </Flex>
+
+    <Flex marginBottom="spacingS">
+      <SectionHeading element="h3">Dropdown with full width</SectionHeading>
     </Flex>
 
     <Dropdown
       isOpen
-      isAutoalignmentEnabled
-      usePortal
-      position="bottom-left"
+      isFullWidth
```

Comment:
It's not defaulting to open in Chromatic 🤔

Reply:
The story above it is open. Could it be because of `usePortal`?

Reply:
do I need to add `usePortal` to make it work?

Reply:
Not sure, can you try it?

Reply:
Just pushed it

Reply:
Doesn't work 🙁 Maybe it's the `position="bottom-left"`?

Reply:
uhmmmm it didn't work, I think chromatic is ignoring the dropdown list because it's absolute positioned or something

Reply:
oops I sent my message before reading yours

Reply:
yep! the problem was that the dropdown does not "pushes" Chromatic's viewport down does
I pushed an empty element with some padding to make it work hahahahah wdyt?

(I could not wrap the dropdown with the Flex component because it makes the `isFullWidth` not work 🤷 )

Reply:
Good enough for me! 😁